### ISSUE TITLE
Getting error - SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by hvaoc** [ArnoSaine/react-pouchdb#13]
This happens when used with Next.Js + Typescript

I am using react-pouchdb like this

```
                        <PouchDB name={this.props.dbName}>
                            <Suspense fallback="loading...">
                                <MyComponent />
                            </Suspense>
                        </PouchDB>
```

Error details:

```
src/node_modules/@postinumero/map-get-with-default/index.js:1
export default function getWithDefault(key, getDefault) {
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/project/test/src/node_modules/ts-node/src/index.ts:807:44)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/project/test/src/node_modules/react-pouchdb/src/utils/createStore.js:1:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/project/test/src/node_modules/ts-node/src/index.ts:807:44)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)

```

**Comment by ArnoSaine** [comment]
Hi, thank you for reporting this issue. One of the dependencies (that I maintain) uses ES module syntax. Seems that it's not supported in Next.js.

Could you check if adding this:
```
"type": "module",
"main": "./index.js",
```
to `src/node_modules/@postinumero/map-get-with-default/package.json` fixes it?

Of course that is not a permanent solution. If it works on your machine, I'll fix that in the package and publish a new version. Let me know 🙂

If that does not work, could you try this?:
https://github.com/martpie/next-transpile-modules

**Comment by functionq** [comment]
I don't know if it is the best way to fix this. But since the dep is small enough, I moved the code inside the createStore.js file itself. 

https://github.com/hvaoc/react-pouchdb/commits/master
https://github.com/hvaoc/react-pouchdb/commit/1d5bfd0cc517ccafdc4ceb651981e2f2c6807856

**Comment by hvaoc** [comment]
I don't know if it is the best way to fix this.

Since the dep is small enough, I moved the code to CreateStore.js.

https://github.com/hvaoc/react-pouchdb/commit/1d5bfd0cc517ccafdc4ceb651981e2f2c6807856

**Comment by ArnoSaine** [comment]
Thanks, I published this fix in v2.0.1 :)
